### DISCUSSION TITLE
Has anyone experienced responses that turn into just #?

### DISCUSSION THREAD

**Discussion by nickanderson** [karthink/gptel#131] [Q&A]
I am seeing some weird responses, I think I see it mostly with zephyr via ollama, at least that seems to be the only place I see it so far. I haven't seen it when I try sending queries with ollama run, only when sent from gptel.

I don't believe I have seen it in a first response. And of course, when I try now I can't seem to trigger it.

I was thinking perhaps it had to do with prompt templates, but it seems like ollama deals with the prompt template?

**Comment by karthink** [comment]
This could be a response parsing bug in gptel. A good start would be to `(setq gptel--debug t)` and see if the response from the model matches what gptel writes into the buffer.
